.github/workflows/autobuild.yml

```diff
@@ -436,20 +437,6 @@ jobs:
         env:
           GITHUB_TOKEN:             ${{ secrets.GITHUB_TOKEN }}
 
-      ## RELEASE PROCEDURE FOR: macOS App Store - storesigned pkg
```

Comment:
Seems as if notarization is not needed for app store submission - at least it fails. Therefore, removing this.